Plugin 3.0.0 requires Java 11 - incorrectly?

**Describe the bug**
The latest version (3.0.0) of this plugin appears to require Java 11.
However as it's claimed to support Sonar 8.9+ where Java 8 is still supported for the scanners: https://docs.sonarqube.org/8.9/requirements/requirements/#header-3 this requirement seems incorrect.
Our Maven projects that use Java 8 and this plugin are now getting an error about the plugin being compiled for Java 11.
We had to revert to the plugin 2.0.8 for now.

**Current behavior**
Maven output:
`[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.9.1.2184:sonar (default-cli) on project xyz: The plugin [dependencycheck] does not support Java 1.8.0_322: org/sonar/dependencycheck/DependencyCheckPlugin has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0`

**Versions (please complete the following information):**
-   sonarqube 8.9.7 @ java 11.0.14
-   dependency-check-sonar-plugin 3.0.0
-   maven 3.6.3 @ java 8u322
-   sonar-maven-plugin 3.9.1.2184

--- a/.github/workflows/testing.yml
+++ b/.github/workflows/testing.yml
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ '11' ]
+        java: [ '8', '11' ]
     name: Java ${{ matrix.Java }} testing
     steps:
       - name: Checkout

--- a/pom.xml
+++ b/pom.xml
@@ -18,7 +18,6 @@
         <license.owner>dependency-check</license.owner>
         <license.mailto>philipp.dallig@gmail.com</license.mailto>
 
-        <jdk.min.version>11</jdk.min.version>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
         <sonar.projectKey>dependency-check_dependency-check-sonar-plugin</sonar.projectKey>
         <sonar.organization>dependency-check</sonar.organization>

--- a/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/base/DependencyCheckConstants.java
+++ b/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/base/DependencyCheckConstants.java
@@ -24,13 +24,13 @@ public final class DependencyCheckConstants {
     /**
      * @deprecated Please use the JSON Report
      */
-    @Deprecated(since = "3.0.0", forRemoval = true)
+    @Deprecated
     public static final String XML_REPORT_PATH_PROPERTY = "sonar.dependencyCheck.xmlReportPath";
 
     /**
      * @deprecated Please use the JSON Report
      */
-    @Deprecated(since = "3.0.0", forRemoval = true)
+    @Deprecated
     public static final String DEPRECTED_XML_REPORT_PATH_PROPERTY = "sonar.dependencyCheck.reportPath";
     public static final String JSON_REPORT_PATH_PROPERTY = "sonar.dependencyCheck.jsonReportPath";
     public static final String HTML_REPORT_PATH_PROPERTY = "sonar.dependencyCheck.htmlReportPath";
@@ -50,7 +50,7 @@ public final class DependencyCheckConstants {
     /**
      * @deprecated Please use the JSON Report
      */
-    @Deprecated(since = "3.0.0", forRemoval = true)
+    @Deprecated
     public static final String XML_REPORT_PATH_DEFAULT = "${WORKSPACE}/dependency-check-report.xml";
     public static final String JSON_REPORT_PATH_DEFAULT = "${WORKSPACE}/dependency-check-report.json";
     public static final String HTML_REPORT_PATH_DEFAULT = "${WORKSPACE}/dependency-check-report.html";

--- a/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/parser/XMLReportParserHelper.java
+++ b/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/parser/XMLReportParserHelper.java
@@ -32,7 +32,7 @@ import com.fasterxml.jackson.dataformat.xml.XmlMapper;
 /**
  * @deprecated Please use the JSON Report
  */
-@Deprecated(since = "3.0.0", forRemoval = true)
+@Deprecated
 public class XMLReportParserHelper {
 
     private XMLReportParserHelper() {

--- a/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/report/ReportFormat.java
+++ b/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/report/ReportFormat.java
@@ -20,7 +20,7 @@
 package org.sonar.dependencycheck.report;
 
 public enum ReportFormat {
-    @Deprecated(since = "3.0.0", forRemoval = true)
+    @Deprecated
     XML,
     HTML,
     JSON

--- a/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/report/XmlReportFile.java
+++ b/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/report/XmlReportFile.java
@@ -30,7 +30,7 @@ import org.sonar.dependencycheck.base.DependencyCheckConstants;
 /**
  * @deprecated Please use the JsonReportFile
  */
-@Deprecated(since = "3.0.0", forRemoval = true)
+@Deprecated
 public class XmlReportFile extends ReportFile {
 
     public static XmlReportFile getXmlReport(Configuration config, FileSystem fileSystem, PathResolver pathResolver) throws FileNotFoundException {

--- a/sonar-dependency-check-plugin/src/test/java/org/sonar/dependencycheck/DependencyCheckSensorTest.java
+++ b/sonar-dependency-check-plugin/src/test/java/org/sonar/dependencycheck/DependencyCheckSensorTest.java
@@ -236,7 +236,7 @@ class DependencyCheckSensorTest {
     }
 
     @Test
-    @Deprecated(since = "3.0.0", forRemoval = true)
+    @Deprecated
     void shouldAddWarningsWithXMLReportPlugin() {
         final SensorContextTester context = SensorContextTester.create(new File(""));
         // Plugin Configuration

--- a/sonar-dependency-check-plugin/src/test/java/org/sonar/dependencycheck/parser/XMLReportParserHelperTest.java
+++ b/sonar-dependency-check-plugin/src/test/java/org/sonar/dependencycheck/parser/XMLReportParserHelperTest.java
@@ -41,7 +41,7 @@ import org.sonar.dependencycheck.parser.element.Analysis;
 import org.sonar.dependencycheck.parser.element.Dependency;
 import org.sonar.dependencycheck.parser.element.Vulnerability;
 
-@Deprecated(since = "3.0.0", forRemoval = true)
+@Deprecated
 class XMLReportParserHelperTest extends ReportParserTest {
 
     @Override

